integrations/azure_ai_search/README.md

```diff
@@ -19,8 +19,7 @@ pip install azure-ai-search-haystack
 ```
 
 ## Examples
-You can find a code example showing how to use the Document Store and the Retriever in the documentation or in [this Colab](https://colab.research.google.com/drive/1YpDetI8BRbObPDEVdfqUcwhEX9UUXP-m?usp=sharing).
-
+You can find a code example showing how to use the Document Store and the Retriever in the documentation.
```

Comment:
we can maybe also link to the `example` directory, where I see there are some examples